examples/exercises/plot_cv_diabetes.py

```diff
@@ -33,10 +34,16 @@
 clf.fit(X, y)
 scores = clf.cv_results_["mean_test_score"]
 scores_std = clf.cv_results_["std_test_score"]
+
 plt.figure().set_size_inches(8, 6)
 plt.semilogx(alphas, scores)
+plt.xlabel("alphas")
+plt.ylabel("scores")
+plt.show()
 
-# plot error lines showing +/- std. errors of the scores
+# %%
+# Plot error lines showing +/- std. errors of the scores
+# ------------------------------------------------------
```

Comment:
```suggestion
```
Sorry for the misunderstanding, maybe these suggested changes make more sense as they will make a single plot of the error line with the corresponding error bands